user_guide_src/source/installation/upgrade_430.rst

```diff
@@ -156,6 +156,7 @@ Others
 Breaking Enhancements
 *********************
 
+- Since void HTML element (e.g. ``<input>``) in ``html_helper``, ``form_helper`` or common functions has been changed to HTML5 valid by default. If you need to be compatible with XHTML, set the ``$html5`` property in **app/Config/DocTypes.php** to ``false``.
```

Comment:
```suggestion
- Since void HTML elements (e.g. ``<input>``) in ``html_helper``, ``form_helper`` or common functions have been changed to be HTML5-compatible by default and you need to be compatible with XHTML, you must set the ``$html5`` property in **app/Config/DocTypes.php** to ``false``.
```

Reply:
Done